DataTypes/Program.cs

```diff
@@ -1,12 +1,145 @@
 ﻿using System;
+using System.Threading;
 
 namespace DataTypes
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string ch1 = "";
+            string ch2 = "";
+            string ch3 = "";
+
```

Comment:
If you set int stick up here, you'll be able to access it throughout the program, and could simplify some of the code. For example:
 if (ch1 == "y")
+            {
+                stick++;
+                Console.WriteLine("You have taken the stick. You now have {0} stick.", stick);
+                Thread.Sleep(2000);
// Stick Not Taken
+            else
+            {
+                Console.WriteLine("You did not take the stick.");
+            }